lib/chart/line.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Chart.Line do
+  @moduledoc """
+  A line chart definition structure
+  """
+
+  alias Chart.Line.{Settings, Svg}
+  alias Chart.Chart
+
+  def put(%Chart{settings: %{storage: storage}} = chart, data) do
+    chart
+    |> storage.put(data)
+    |> Chart.put_data(data)
```

Comment:
Neni divne, ze zde se ukladaji data a pak se ukladaji i v `buffer.ex`?

Reply:
jo vidis, ja uz si to vcera nestihal zkontrolovat...